assets/js/components/Work/UIWorkListItem.jsx

```diff
@@ -1,48 +1,62 @@
-import React from "react";
+import React, { useContext } from "react";
 import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
 import { setVisibilityClass, formatDate } from "../../services/helpers";
+import { IIIFProvider, IIIFContext } from "../../components/IIIF/IIIFProvider";
 
-const WorkListItem = ({ work }) => {
+const WorkListItem = ({
+  id,
+  representativeImage,
+  title,
+  descriptiveMetadata,
+  workType,
+  visibility,
+  published,
+  accessionNumber,
+  fileSets,
+  manifestUrl,
+  updatedAt,
+}) => {
+  const iiifServerUrl = useContext(IIIFContext);
   return (
     <>
-      <article className="media " data-testid="ui-worklist-item">
+      <article className="media" data-testid="ui-worklist-item">
         <figure className="media-left">
           <p
             className="image is-square"
             style={{ width: "250px", height: "250px" }}
           >
-            <Link to={`/work/${work.id}`}>
+            <Link to={`/work/${id}`}>
               <img
                 src={`${
-                  work.representativeImage
-                    ? work.representativeImage + "/full/1280,960/0/default.jpg"
-                    : "/images/1280x960.png"
+                  representativeImage.id
+                    ? iiifServerUrl +
+                      representativeImage.id +
+                      "/square/500,500/0/default.jpg"
+                    : representativeImage + "/full/1280,960/0/default.jpg"
                 }`}
                 data-testid="image-work"
-                alt={work.title}
+                alt={title}
               />
             </Link>
           </p>
         </figure>
         <div className="media-content">
           <h3 className="title is-size-4">
-            {work.descriptiveMetadata.title
-              ? work.descriptiveMetadata.title
-              : "Untitled"}
+            {descriptiveMetadata.title ? descriptiveMetadata.title : "Untitled"}
           </h3>
           <div className="content ">
             <p>
-              <span className="tag">{work.workType.label}</span>
-              {work.visibility && (
+              <span className="tag">{workType.label.toUpperCase()}</span>
+              {visibility && (
                 <span
                   data-testid="tag-visibility"
-                  className={`tag ${setVisibilityClass(work.visibility.id)}`}
+                  className={`tag ${setVisibilityClass(visibility.id)}`}
                 >
-                  {work.visibility.label.toUpperCase()}
+                  {visibility.label.toUpperCase()}
                 </span>
               )}
-              {work.published && (
+              {published && (
                 <span data-testid="dd-published" className="tag is-success">
```

Comment:
For the test ids, the `dd-` prefix could be confusing and might now be out of context.  Let's make it something more generic like `result-item-published`, `result-item-accession-number`, etc.?